src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  &__title {
+    padding-inline: 16px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+
+    position: absolute;
+    top: 206px;
+    transition-duration: $transition;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__product-code {
+    padding-inline: 16px;
+    position: absolute;
+    top: 246px;
+    color: #616070;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+  }
+
+  &__price {
+    padding-inline: 16px;
+    width: 166px;
+    position: absolute;
+    top: 316px;
```

Comment:
Consider using relative units (like percentages or ems) for positioning to improve responsiveness, especially for the `top` property.